jsx/Form.js

```diff
@@ -1752,11 +1752,15 @@ class LorisElement extends Component {
 class RadioElement extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
```

Comment:
i don't think you need to set a state here. you can use this.props.checked directly